submissions/best-of-its-genre-nanogamer7/origins.md

```diff
@@ -0,0 +1,46 @@
+## Best of its genre - Nanogamer7
+
+We've all heard of Factorios insanely [good ratings](https://steamdb.info/stats/gameratings/) - consistently rising to second place after Portal 2 again and again.[^1] Obviously many others will try to imitate this success in the automation genre, but where did Factorio get inspired from? That's what I'll be trying to answer in this first article of hopefully a multi-part series.
```

Comment:
You could add the question "was it the first game solely designed for automation or merely the first one to do it successfully?"

Reply:
I hope it's ok if I'll just copy that question, it fits pretty much perfectly into my text